src/main/java/Duke.java

```diff
@@ -1,10 +1,62 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        // Start chat session
+        ChatSession currChat = new ChatSession();
+        currChat.startSession();
+
+        // Get user input
+        Scanner sc = new Scanner(System.in);
+        String userInput;
+        String[] userInputArr;
+        String userCommand;
+        String[] userArguments;
+        String description;
+
+        while (true) {
+            userInput = sc.nextLine();
+            userInputArr = userInput.split(" ", 2);
+            userCommand = userInputArr[0];
+
+            if (userCommand.startsWith("bye")) {
```

Comment:
all methods, conditionals, and loops seem to follow the K&R style brackets, and statements within are all in the next line so that's great